apps/coscrad-frontend/src/components/dynamic-forms/dynamic-form-elements/multilingual-text-input/multilingual-text-input.tsx

```diff
@@ -49,6 +50,7 @@ const SingleLanguageField = ({
                     required={false}
                 />
                 <Select
+                    data-testid={`role-entry:${languageCode}`}
```

Comment:
This is ideal for testids where possible

Reply:
The problem is MUI doesn't always flow the data attributes through. No point of doing it this way if your test doesn't pass.